draft-ietf-jsonpath-base.md

```diff
@@ -1276,7 +1276,7 @@ The comparison operators `==` and `<` are defined first and then these are used
 When a path or function expression resulting in an empty nodelist appears on either side of a comparison:
 
 * a comparison using the operator `==` yields true if and only if the comparison
-is between two expressions, each of which is a path or a function expression, and each of which result in an empty nodelist.
+is between two `comparable`s, each of which is a path or a function expression, and each of which result in an empty nodelist.
 
 * a comparison using the operator `<` yields false.
```

Comment:
Editorial:  Does this mean that `>=` yields true?  Can be solved later.

Reply:
We have defined `>=` as `==` || `>`, where `>` is `swap and <`.
Yes, it hurts that `>=` ≠ `! <`, but that's where we are.

Reply:
I actually think `swap and <` is good.

`"foo" < 4` should yield false because they're incompatible.  Similarly, `"foo" >= 4` should also yield false for the same reason.  Replacing `"foo"` with `Nothing` should give the same results for the same reasons.

Conversely, if the operands are compatible then `>=` does imply `! <`.

This is set up right.